src/FsLibLog/FsLibLog.fs

```diff
@@ -789,3 +789,54 @@ module LogProvider =
         let stackFrame = StackFrame(2, false)
         getLoggerByType(stackFrame.GetMethod().DeclaringType)
 
+/// Provides operators to make writing logs more streamlined.
+module Operators =
+
+    /// Initializes a log with the specified message
```

Comment:
please don't use the old markdown-style comments, it just causes pain for non-VS Code editors :(  Ionide should be able to generate properly-formatted xmldoc blocks for you these days though to help make the process smoother.